src/sentry/static/sentry/app/components/todos.jsx

```diff
@@ -0,0 +1,312 @@
+import moment from 'moment';
+import React from 'react';
+import {t} from '../locale';
+
+import ApiMixin from '../mixins/apiMixin';
+import OrganizationState from '../mixins/organizationState';
+
+const TodoItem = React.createClass({
+  propTypes: {
+    task: React.PropTypes.object,
+  },
+
+  mixins: [OrganizationState],
+
+  getInitialState: function() {
+    return {
+      showConfirmation: false
+    };
+  },
+
+  toggleConfirmation: function() {
+    this.setState({showConfirmation: !this.state.showConfirmation});
+  },
+
+  skip: function(task) {
+    this.props.onSkip(task);
+    this.setState({showConfirmation: false});
+  },
+
+  render: function() {
+    let org = this.getOrganization();
+
+    let classNames = '';
+    let description = '';
+    let doneDescription = 'By being here, you\'ve done it. Welcome to Sentry!';
+    if (this.props.task.status == 'Complete') {
+      classNames += ' checked';
+
+      if (this.props.task.task == 0) {
+        description = doneDescription;
+      } else {
+        description = this.props.task.user + ' completed this ' + moment(this.props.task.date_completed).fromNow();
+      }
+
+    } else if (this.props.task.status == 'Pending') {
+      classNames += ' pending';
+      description = this.props.task.user + t(' kicked this off ') + moment(this.props.task.date_completed).fromNow();
```

Comment:
These strings are not translated properly as-is. Translatable strings need the full context (user, timestamp) – here a translator only sees "kicked this off".

It's probably okay to punt on translations until onboarding is 100% done (i.e. do a pass on them later, since we're not going to translate them immediately).

cc @mitsuhiko